src/components/builder.ts

```diff
@@ -56,6 +58,12 @@ export class Builder {
         this.extension.logger.displayStatus('sync~spin', 'statusBar.foreground')
         this.preprocess(rootFile)
 
+        this.pageTotal = undefined
+        // @ts-ignore
+        pdfjsLib.getDocument(rootFile.replace(/\.tex$/, '.pdf')).promise.then(doc => {
```

Comment:
This needs to be 

```
pdfjsLib.getDocument(this.extension.manager.tex2pdf(rootFile, true)).promise.then(doc => {
```

 in order to work with `outDir` specified.

Reply:
Applied in 0a24c2c32033d2920b37254642988e5004bf213d